rails-questions.md

```diff
@@ -1,30 +1,33 @@
 # ASSESSMENT 5: Database and Active Record Practical Questions
 
 1. What is SQL?
-
+SQL stands for structured query language and allows for communication between programmers and databases.
 
 2. What the PostgreSQL query that would return all the content in a particular table?
+SELECT * ALL
+FROM <table>
 
 
 3. What is the command to create a new Rails application with a PostgreSQL database?
-
+rails create new <app_name>
 
 4. What is the command to generate a Rails model for a meals table with columns named breakfast, lunch, and dinner?
-
+rails g model Meals breakfast:string lunch:string dinner:string
 
 5. What is a migration? Why would you use one?
-
+Migration allows you to manipulate/update/alter information in a database, you might use a migration to change the name of a column.
```

Comment:
Migration allows you to manipulate/update/alter _**the*_* database, you might use a migration to change the name of a column **_or add a table or manipulate the values of data that can be stored_**.